taxonomy-editor-frontend/src/pages/editentry/ListAllOtherProperties.jsx

```diff
@@ -1,22 +1,97 @@
-import { Box, Typography } from "@mui/material";
+import { Box, Paper, Stack, TextField, Typography } from "@mui/material";
+import ISO6391 from 'iso-639-1';
 
 // Parent component used for rendering info
 // on a stopword, synonym, header or footer
 
 const ListAllOtherProperties = ({ nodeObject, id, setNodeObject }) => {
+    
+    // Stores 2 letter language code (LC) of the tags
+    let languageCode = ''; 
+    // Storing keys and values that needs to be rendered for editing
+    let renderedOtherProperties = {} 
+
+    if (nodeObject) {
+        Object.keys(nodeObject).forEach((key) => {
+
+            // Get all tags and its corresponding language code
+            // Tagids need to be recomputed, so shouldn't be rendered
+
+            if (key.startsWith('tags') && 
+                !key.includes('ids')) {
+                    languageCode = key.slice(-2);
```

Comment:
mmm, maybe I'm missing something, but, it is redifining the `let languageCode` on every `key` of the `nodeObject`... it seems a bit unpredictable what would be the final value, makes me wonder if all have the same value and 1) what happens if so and 2) what happens if that's not the case ...

Reply:
@nobeeakon Yeah, you're right. I had actually thought about this, and wrapped the `Object.keys()` function inside a `useEffect` combined with state variables for `languageCode`. This will come in an another PR.

Reply:
I'll add a TODO to this to remind us that this will be changed in another PR.